### DISCUSSION TITLE
Help regarding running this on my privacy tweaked windows 11 21h1

### DISCUSSION THREAD

**Discussion by ghost** [ALTaleX531/TranslucentFlyouts#30] [Q&A] [answered]
Edit: Just found out the flyout is present in Task Scheduler and it is set to disable on unplugging power on my laptop but it was an easy fix but is there any tweak in task scheduler to make this more permanent?

It worked once and itself downloaded symbols after restart and I absolutely loved how the ui looked. After a restart it didnt work. I tried everything to get it working but it slighly works in a limited places right after installation and a restart reverts everything back. It also doesnt seem to be on the registry....Please someone help me get this activated permanently 

The reason I am still on 21h1 is that 22h2 performs incredibly worse on my Acer Nitro 5 2022 model even after all drivers and bios update for that specific issue. But I even enabled windows update for this so I wonder what are other factors required for this to work flawlessly.

**Comment by ALTaleX531** [comment, marked as answer]
If the registry keys or values are not exist, then you have to manually create them by yourself. I think I will solve the Task Scheduler issue in the next release...

**Reply by ghost** [reply]
Is the issue that I'm facing a unique case or few more people face this too? 

Eventhough I changed few settings in task scheduler it still turns off sometimes after disconnnecting external monitor and I have to manually run it. Can you recommend any settings so its permanent?

Thank you for this beautiful tweak....now the old flat color looks boring....I cant go back.....looking forward to the future bug fixes and updates.
